src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -60,6 +62,16 @@ func New(location, namespace, resource string) Dumper {
 				"perconaxtradbclusterbackups",
 				"perconaxtradbclusterrestores",
 				"perconaxtradbclusters")
+			filePaths = append(filePaths,
+				"var/lib/mysql/mysqld-error.log",
+				"var/lib/mysql/innobackup.backup.log",
+				"var/lib/mysql/innobackup.move.log",
+				"var/lib/mysql/innobackup.prepare.log",
+				"var/lib/mysql/grastate.dat",
+				"var/lib/mysql/gvwstate.dat",
+				"var/lib/mysql/mysqld.post.processing.log",
+				"var/lib/mysql/auto.cnf",
```

Comment:
any reason to not use absolute paths? currently we don't set `workingDir` in operators but if it's set, this will break.

Reply:
Because they are taken few times, for each of PXC nodes: cluster1-pxc-0, cluster1-pxc-1, and cluster1-pxc-2. So I have to dynamically create final absolute path.

Reply:
what worries me is we're using these paths as they're with `kubectl cp ...` and this can break with a `workingDir` field in PXC containers. but I don't think it's a problem right now, just room for improvement.